src/components/appSwitch/AppSwitch.js

```diff
@@ -232,21 +232,6 @@ const AppSwitch = (props: LoadingProps) => {
 
   useEffect(() => {
     initWalletAndStorage(undefined, 'SEED', true).then(() => log.debug('storage and wallet ready'))
-  }, [])
```

Comment:
revert this

Reply:
1. you're missing callback actual value (by removing use effect updating the ref
2. initWalletAndStorage is async, you have to subscribe only if it's done

prevous implelemntations should work well

Reply:
1. navigateToUrlAction isn't a callback and doesn't have outer dependancies
2. ok